src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER = 100;
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+
+        return new Ball(colorSupplier.getRandomColor(), random.nextInt(MAX_NUMBER));
```

Comment:
to increase readability and for better debugging, create local variables for random color and random number